components/base/src/test/resources/META-INF/services/pl.touk.nussknacker.engine.flink.api.exception.FlinkEspExceptionConsumerProvider

```diff
@@ -0,0 +1 @@
+pl.touk.nussknacker.engine.flink.test.RecordingExceptionConsumerProvider
```

Comment:
let's put it in flink-test-util?

Reply:
:+1:

Reply:
https://github.com/TouK/nussknacker/pull/2484/commits/dd2818b975a391c5ddc42f2335c8be2b316042fe